contributed_definitions/NXspm.nxdl.xml

```diff
@@ -501,6 +467,11 @@
             <doc>
                 The sample information.
             </doc>
+            <group type="NXsample_component_set" optional="true">
```

Comment:
Shouldnt that base class get removed/or wasnt that already removed?

Reply:
NXsample_component_set is still available and I support that base class. specially whey we will have multiple layers of material and substrate.

https://github.com/FAIRmat-NFDI/nexus_definitions/blob/fairmat/contributed_definitions/NXsample_component_set.nxdl.xml

Reply:
Okay, yeah that is fine then for the time begin